lib/settings.js

```diff
@@ -35,9 +36,15 @@ async function plistPaths (sim, identifier) {
       paths.push(path.resolve(sim.getDir(), 'Library', 'Preferences', 'com.apple.locationd.plist'));
       break;
     case 'userSettings':
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'UserSettings.plist'));
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'EffectiveUserSettings.plist'));
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'PublicInfo', 'PublicEffectiveUserSettings.plist'));
+      if (compare(sim.xcodeVersion.versionString, '7.3') < 0) {
```

Comment:
It might worthwhile to get the data directory once and then use a local variable. Seems like a lot of `sim.getDir()` (though I know that has always been there).